igdbHelper.js

```diff
@@ -0,0 +1,70 @@
+require("dotenv").config();
+const axios = require("axios");
+
+const { CLIENT_ID, CLIENT_SECRET } = process.env;
+
+let tokenExperationTime;
+let accessToken;
+
+//sets token and timer, if token doesn't exist and/or is expired.
+const getAccessToken = async () => {
+  const currentTime = new Date().getTime();
+  if (accessToken && currentTime < tokenExperationTime) return;
+
+  const url = `https://id.twitch.tv/oauth2/token?client_id=${CLIENT_ID}&client_secret=${CLIENT_SECRET}&grant_type=client_credentials`;
+  const response = await axios.post(url);
+
+  accessToken = response.data.access_token;
+  tokenExperationTime = response.data.expires_in + currentTime;
+};
+
+const reusableAxiosPost = async (routeName) => {
+  await getAccessToken();
+
+  const config = newReleasesPostConfig();
+  const data = postData[routeName]();
+  return await axios.post("https://api.igdb.com/v4/games", data, config);
+};
+
+module.exports = reusableAxiosPost;
+
+//AXIOS CONFIG
+//axios posts default config
+const newReleasesPostConfig = () => {
```

Comment:
The function naming is weird here. The same config is used for any axios.post call to that api.

Reply:
I meant to change that to match the config. I forgot it was set that way, but the goal was to put something in place incase we had to change any headers. Just got left behind and forgotten in the mess.